pkg/minikube/tests/driver_mock.go

```diff
@@ -40,7 +40,7 @@ func (driver *MockDriver) Create() error {
 }
 
 func (driver *MockDriver) GetIP() (string, error) {
-	return driver.BaseDriver.GetIP()
+	return "192.168.99.107", nil
```

Comment:
Sorry Aaron, I was wrong when we chatted about this. We already have a way to set the IP at a per-test level. See here:
https://github.com/kubernetes/minikube/blob/master/pkg/minikube/cluster/cluster_test.go#L314